specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json

```diff
@@ -726,7 +726,7 @@
               "Monthly"
             ],
             "x-ms-enum": {
-              "name": "aggregationInterval",
+              "name": "latencyScoreboardAggregationInterval",
```

Comment:
this is a potential breaking change, is this a swagger bug fixing to align with existing service behavior?

Reply:
this is fine :)

Reply:
this should say "latencyScorecardAggregationInterval"

Reply:
@minechosoft renamed to Scorecard

@yungezz changing name in **x-ms-enum** only results in SDK changes. anyway, we had to change this name, as there were two different enums with the same name, and that was causing autorest to fail. Any other changes I made were checked by @minechosoft and service is going to be aligned.